src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -539,6 +541,38 @@ public void staticUserExport(OutputStream output, boolean convertHeader) throws
         }
     }
 
+    void userWithTeamsExport(OutputStream output) throws IOException{
+        log.info("Exporting USER_TEAMS_DATA to CSV", value(EVENT, CSV_EXPORT_START));
+        OutputStream buffer = new BufferedOutputStream(output);
+        OutputStreamWriter outputWriter = new OutputStreamWriter(buffer, "UTF-8");
+        List<String> headers = Stream.of("User UUID", "Teams UUID").collect(Collectors.toList());
```

Comment:
This could just be `List.of` to simplify.

```suggestion
        List<String> headers = List.of("User UUID", "Teams UUID");
```

May need to add the import.